### PR TITLE
Add the technical writing resources

### PR DESCRIPTION
**Pull request by motuncoded**
This commit introduces the technical resources section

### CODE REVIEW

File: TechnicalWriting/TechicalWriting.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+### **Doxygen** (Code Documentation)
+![Doxygen](./assets/Doxygen.png)
+
+A tool to automatically generate documentation from annotated source code.
+
+- **[Doxygen Official Docs](https://doxygen.nl/manual/index.html)** - Detailed instructions for using Doxygen in documentation.
+
+### **Hashnode** 
+![](./assets/Hashnode.png)
```
Comment (Charity10) [inline TechnicalWriting/TechicalWriting.md:83]: Add the hashnode text here

**Review by Charity10 — CHANGES_REQUESTED** [review]
Please update the readme file.

**Comment by motuncoded** [comment]
Would be glad to do that

File: README.md [hunk 2]
```diff
@@ -10,8 +10,10 @@ Welcome to the **Learning Resources Repository**! This repository is designed to
 2. [Backend Development](#-backend-development)
 3. [Mobile Development](#mobile-development)
 4. [Personal Development](#-personal-development)
-5. [Spatial Computing](#spatial-computing)
+5. [Spatial Computing](#-spatial-computing)
 6. [Machine Learning](#-machine-learning)
+7. [Technical Writing](#-technical-writing)
+
```
Comment (Charity10) [inline README.md:16]: No 5 and 7 links not working
For no 5  just pull from the master to get the updated version, Arafat already fixed that

**Comment by motuncoded** [comment]
Ok

**Pull request merged**